src/components/Profile.js

```diff
@@ -127,23 +305,6 @@ class Profile extends Component {
                   </div>
                 </div>
               </div>
-
-              <h4 className="mt-5 mb-2">Deine Sammlungen</h4>
-              {this.state.profileUser.userId === this.props.registeredUserId ? (
-                <p>
-                  <Link to="/add-recipe" className="nav-link addButton">
-                    <FiPlusCircle /> Neue hinzufügen
-                  </Link>
-                </p>
-              ) : null}
-
-              <div className="row">
-                <CollectionPreview />
-                <CollectionPreview />
-                <CollectionPreview />
-                <CollectionPreview />
-              </div>
-
               <h4 className="mt-5 mb-2">Deine Rezepte</h4>
```

Comment:
```suggestion
              <h4 className="mt-5 mb-2">Your Recipes</h4>
```
Das wäre das einzig deutsche in unserer Anwendung 🙃